src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -282,6 +282,35 @@ private void testEditQuestionAction() throws Exception {
                                 .contains("The recipient's team members can see your response, but not the name of the recipient, or your name."));
         
         feedbackEditPage.clickAndCancel(feedbackEditPage.getCancelQuestionLink(-1));
+        
+        ______TS("add question 2 and edit it to giver's team members and giver");
+        feedbackEditPage.clickAddQuestionButton();
+        feedbackEditPage.fillQuestionBox("test visibility when choosing giver's team members and giver");
+        feedbackEditPage.selectGiverToBeStudents();
+        feedbackEditPage.selectRecipientsToBeGiverTeamMembersAndGiver();
+        feedbackEditPage.clickMaxNumberOfRecipientsButton();
+        feedbackEditPage.clickAddQuestionButton();
+        
+        ______TS("test Recipient's Team Members row is hidden");
+        feedbackEditPage.clickVisibilityOptionsForQuestion(2);
+        
+        assertEquals("Recipient's Team Members", feedbackEditPage.getVisibilityOptionTableRow(2, 4).getAttribute("textContent").trim());
+        assertFalse(feedbackEditPage.getVisibilityOptionTableRow(2, 4).isDisplayed());
+                
+        ______TS("test visibility preview of question 2");
+        feedbackEditPage.clickVisibilityPreviewForQuestion(2);
+        WebElement visibilityMessage2 = browser.driver.findElement(By.id("visibilityMessage-2"));
+        feedbackEditPage.waitForElementVisibility(visibilityMessage2);
+
+        assertTrue("Expected recipient to be able to see response, but was "
```

Comment:
@Sumei1009, we can specify the recipient type like in the other similar tests above, instead of just <code>Expected recipient ... </code>